specs/PushNotifications/PushNotifications-spec.md

```diff
@@ -318,7 +388,10 @@ else if (result.Status() == ChannelStatus::CompletedFailure)
 ```
 
 ## Manifest Registration
-For MSIX, the COM activator GUID and the exe path need to be registered in the manifest. The launch args would need to be pre set to a well-known string that defines Push Triggers.
+
+For MSIX, the COM activator GUID and the exe path need to be registered in the manifest. The launch
```

Comment:
NIT: ...and the exe need to be...

One specifies an exe in the manifest (as a relative path/filename) so you don't really specify the path in the manifest (some manifest features do let you specify an actual/full path, but I don't think this is one of them)